db/schema.rb

```diff
@@ -307,7 +309,6 @@
     t.string "slug"
     t.string "status", default: "active"
     t.datetime "updated_at", null: false
-    t.index ["slug"], name: "index_chat_channels_on_slug", unique: true
```

Comment:
I think this shouldn't be in this PR?

Reply:
Yeah these somehow found their way in 🤷‍♀️ They weren't my changes. I will fix.